docker-compose.yml

```diff
@@ -16,7 +10,6 @@ services:
         - eu.gcr.io/tmrow-152415/electricitymap_web:latest
     image: eu.gcr.io/tmrow-152415/electricitymap_web:latest
     command: npm run server-dev
-    depends_on: ['mockserver']
```

Comment:
could i ask why we don't need this anymore?

Reply:
We want to simplify things to speed up and eventually get away from having docker as a prerequisite, so this is taking one step further as we are able to run the mockserver without docker (and running the map without the mockserver) :)